scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -816,12 +841,39 @@ def _get_binaries(self) -> List[str]:
             ]
         return binaries
 
-    def _get_binaries_from_runners(self) -> List[str]:
+    def _get_domains(self) -> List[str]:
+        """
+        Get list of domains from domains.yml (if present) which is generated
+        during a sysbuild.
+        """
+
+        domains: List[str] = []
+
+        domains_file_path: str = os.path.join(self.instance.build_dir, 'domains.yaml')
```

Comment:
Shouldn't this just check the test yml file to see if `sysbuild: true` is present rather than guessing if a file is present or not?